addons/zylann.sdf_blender/sdf_item.gd

```diff
@@ -99,8 +89,9 @@ func _notification(what: int):
 		# TODO Visibility?
```

Comment:
I believe `NOTIFICATION_TRANSFORM_CHANGED` is not occurring, which is why I had to manually set the transform at `_ready()`. This will prevent any run-time transform changes to work.